config/config-builder/src/main.rs

```diff
@@ -102,39 +111,54 @@ struct ValidatorArgs {
     #[structopt(short = "b", long, parse(from_str = parse_addr))]
     /// Advertised address for the first node in this test net
     bootstrap: Multiaddr,
+    #[structopt(short = "l", long, parse(from_str = parse_addr))]
+    /// Listening address for this node
+    listen: Multiaddr,
+    #[structopt(short = "t", long, parse(from_os_str))]
+    /// Path to a template NodeConfig
+    template: Option<PathBuf>,
+    #[structopt(flatten)]
+    validator_common: ValidatorCommonArgs,
+}
+
+#[derive(Debug, StructOpt)]
+struct ValidatorCommonArgs {
     #[structopt(short = "d", long, parse(from_os_str))]
     /// The data directory for the configs (e.g. /opt/libra/etc)
     data_dir: PathBuf,
     #[structopt(short = "i", long, default_value = "0")]
     /// Specify the index into the number of nodes to write to output dir
     index: usize,
-    #[structopt(short = "l", long, parse(from_str = parse_addr))]
-    /// Listening address for this node
-    listen: Multiaddr,
-    #[structopt(short = "n", long, default_value = "1")]
-    /// Specify the number of nodes to configure
-    nodes: usize,
     #[structopt(short = "o", long, parse(from_os_str))]
     /// The output directory
     output_dir: PathBuf,
+    #[structopt(short = "n", long, default_value = "1")]
+    /// Specify the number of nodes to configure
+    nodes: usize,
+    #[structopt(long, parse(from_str = parse_socket_addr))]
+    /// Specify the IP:Port for Safety rules. If this is not defined, SafetyRules will run in its
+    /// default configuration.
+    safety_rules_addr: Option<SocketAddr>,
```

Comment:
what's the default configuration?

Reply:
don't modify safety rules which will result in the default defined in config/src/config/safety_rules.rs which is on disk and local.